plugins/jenkins/app/helpers/jenkins_helper.rb

```diff
@@ -29,17 +31,16 @@ def jenkins_status_panel(deploy, jenkins_job)
       jenkins_job.update_attributes!(attributes)
     end
 
-    if jenkins_job_status.include? ' ' # since it has whitespace, it's probably a human-readable message
+    if status = JENKINS_MAPPING[jenkins_job_status]
+      status_message = JENKINS_RESULT.fetch(jenkins_job_status)
+    else
       status = 'warning'
       status_message = jenkins_job_status
-    else
-      status = mapping.fetch(jenkins_job_status)
-      status_message = result.fetch(jenkins_job_status)
     end
 
     content = "Jenkins build #{jenkins_job.name} for #{deploy.stage.name} #{status_message}"
-    content_tag :a, href: jenkins_job_url, target: "_blank" do
-      content_tag :div, content.html_safe, class: "alert alert-#{status}"
```

Comment:
@rdhanoa does this need to include html that actually gets parsed or can it be escaped?